cmd/client.go

```diff
@@ -156,24 +161,27 @@ func runClient() {
 		default:
 		}
 
-		err := ghClient.TriagePRFailures(globalCtx, jc, flakeCfg, prNumber, urlFails, issueKnownFlakes, jobNameToJenkinsFails, triggerRegexp)
+		err := ghClient.TriagePRFailures(globalCtx, jc, cfg.FlakeTracker, prNumber, urlFails, issueKnownFlakes, jobNameToJenkinsFails, triggerRegexp)
 		if err != nil {
 			panic(err)
 		}
 	}
 }
 
-func loadConfig(cfgFile string) (*github.FlakeConfig, error) {
+func loadConfig(cfgFile string) (*github.PRBlockerConfig, error) {
 	b, err := ioutil.ReadFile(cfgFile)
 	if err != nil {
 		return nil, err
 	}
 
-	var cfg github.FlakeConfig
+	var cfg github.PRBlockerConfig
```

Comment:
@aanm is this breaking the current client mode workflow? I guess the FlakeConfig has now to be under `flake-tracker:`, is that a problem?

Reply:
It's fine, client mode use meant for development purposes anyway.